Public/Src/Deployment/NugetPackages.dsc

```diff
@@ -8,17 +8,18 @@ import * as Managed from "Sdk.Managed.Shared";
 import * as Nuget from "Sdk.Managed.Tools.NuGet";
 
 namespace NugetPackages {
-    export declare const qualifier : { configuration: "debug" | "release"};
+
+    export declare const qualifier : { configuration: "debug" | "release" };
 
     const packageNamePrefix = BuildXLSdk.Flags.isMicrosoftInternal
         ? "BuildXL"
         : "Microsoft.BuildXL";
+
     const packageTargetFolder = BuildXLSdk.Flags.isMicrosoftInternal
         ? r`${qualifier.configuration}/pkgs`
         : r`${qualifier.configuration}/public/pkgs`;
 
-
-    const net472 = pack({
+    const net472 = Context.getCurrentHost().os === "macOS" ? undefined : pack({

```

Comment:
>t.getCurrentHost().os === "macO [](start = 25, length = 31)

perhaps add a local boolean shouldProduceAllNugetPacakges. Reason is that the check here is NOT the inverse of the check on line 113. If we add a third OS we'll end up with a funny split. so this should be always `=== "win"` and `!== "win"` or  `"macOS"`, but not `=== "win"` on one and `=== "macOs"` on the other :D

Reply:
Yeah, will refactor this, its a WIP in the PR title 💃